tortoise/algorithm.go

```diff
@@ -49,24 +52,32 @@ func (alg *Algorithm) HandleLateBlock(b *types.Block) (types.LayerID, types.Laye
 	return oldPbase, newPbase
 }
 
-func (alg *Algorithm) PersistTortoise() error {
-	alg.Lock()
-	defer alg.Unlock()
+//Persist saves a copy of the current tortoise state to the database
+func (alg *tortoise) Persist() error {
```

Comment:
maybe rename alg to trtl or something like that?

Reply:
good idea